docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracking-events/index.md

```diff
@@ -964,12 +964,58 @@ This is a function used to determine which elements are tracked. The element is
 
 This is a function used to transform data in each form field. The value and element (2.15.0+ only) are passed as arguments to the function and the tracked value is replaced by the value returned.
 
+**In versions prior to 3.16.0**, the transform function would receive 2 arguments, that were different between `submit_form` and `change_form` or `focus_form` events. More specifically, the transform function signature **was**:
```

Comment:
Can we put this prior to 3.16 section to a `:::note` admonition? Just to visually separate it from the current behaviour.

So something like
```
:::note Behaviour prior to version 3.16 of the tracker
...
:::
```